metricbeat-tests/Makefile

```diff
@@ -11,5 +16,13 @@ install-godog:
 	go get github.com/DATA-DOG/godog/cmd/godog
```

Comment:
Not related to this PR, but commands installed during CI should be vendorized, or fixed somehow to a version, to avoid failures by unexpected changes.

Reply:
Agreed